README.md

```
@@ -1,42 +1,42 @@
+- [apps/kyc-server](./apps/kyc-server): The KYC server.
+- [apps/contract](./apps/contract): The smart contract.
+- [services/supabase](./services/supabase): The Supabase service.
+- [services/ai](./services/ai): The AI service.
+- [services/dictionary](./services/dictionary): The dictionary service.
+
+## 🚀 Quick Start
+
+### BEFORE ANYTHING ELSE
+
+> **Mac/Linux Users:** Follow the [Taskfile.yml](./Taskfile.yml) for the full list of commands to run the project in different areas. Each command has a description of what it does.
+
```

markdownlint-cli2 (0.17.2)

Blank line inside blockquote
null

(MD028, no-blanks-blockquote)